src/index.ts

```diff
@@ -109,14 +115,9 @@ const getList = <M extends Model>(
 
     const parsedFilter = filter ? parseFilter(filter) : {}
 
-    const { count, rows } = await model.findAndCountAll({
-      offset: from,
-      limit: to - from + 1,
-      order: [sort ? JSON.parse(sort) : ['id', 'ASC']],
-      where: parsedFilter,
-      raw: true,
-    })
-
+    const { rows, count } = parsedFilter['q']
+      ? await search(parsedFilter['q'], to - from + 1, model, searchableFields)
```

Comment:
We may want to keep the sort here (or explain in the doc that search and sort can't co-exist)

Reply:
This does not concern our users because the react-admin dataProvider handle this alone